lib/vbox/vbox.go

```diff
@@ -53,8 +53,18 @@ type (
 		Url  string `json:"url"`
 		Port string `json:"url"`
 	}
+
+	OnOff bool
```

Comment:
__[golint]__
* [golint] exported type OnOff should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->